packages/contracts/test/contracts/libraries/utils/Lib_EthUtils.spec.ts

```diff
@@ -1,4 +1,5 @@
-/* tslint:disable:no-empty */
+/* eslint no-empty-function: "off"*/
+/* eslint @typescript-eslint/no-empty-function: "off" */
```

Comment:
I can't find any empty functions in this file. Do we need this?
If there's only one of them, then let's use a `disable-next-line` instead please. :)

Reply:
Ah the empty functions start around line 303; i could just disable the rule for that block